programs/mango-v4/src/instructions/openbook_v2_place_order.rs

```diff
@@ -136,6 +136,12 @@ pub fn openbook_v2_place_order(
         });
     }
 
+    // Get price lots before the book gets modified
+    let bids = ctx.accounts.bids.load_mut()?;
+    let asks = ctx.accounts.asks.load_mut()?;
+    let order_book = openbook_v2::state::Orderbook { bids, asks };
+    let price_lots = order.price(now_ts, None, &order_book)?.0;
```

Comment:
we do explicitly not allow oracle pegged orders via mango, right? (this None will cause an error)

Reply:
yes, we build the openbook v2 order in lib.rs:1769 from our own types so it's not possible to place oracle pegged orders